src/App.tsx

```diff
@@ -1,14 +1,34 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
 import { TodoModal } from './components/TodoModal';
-import { Loader } from './components/Loader';
+import { ActionTypes } from './types/Actions';
 
 export const App: React.FC = () => {
+  const [error, setError] = useState('');
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const onPageLoad = async () => {
```

Comment:
```suggestion
  const handlePageLoad = async () => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364